src/components/walkthroughResources/walkthroughResources.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const WalkthroughResources = props => {
+  const { resources } = props;
+  let resourceList;
```

Comment:
It might be better to initialize `resourceList` to `null` instead of `undefined`. React can't render undefined, but null is fine: https://github.com/facebook/react/issues/3738

Reply:
TIL, good point.